dcos_test_utils/dcoscli.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import os
+import subprocess
+
+log = logging.getLogger(__name__)
+
+DCOS_CLI_URL = "https://downloads.dcos.io/binaries/cli/linux/x86-64/latest/dcos"
+
+
+class DCOSCLI():
```

Comment:
I know the all-caps is technically PEP8, but I think `DcosCli` is more parseable when skimming the file. Or maybe even just `CLI` since the `Configuration` class below isn't prefixed with DCOS.